server/routes/profile.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  getProfile,
+  getAllProfiles,
+  createProfile,
+  updateProfile,
+} = require("../controllers/profile");
+
+router.route("/").post(protect, createProfile);
+router.route("/update").post(protect, updateProfile);
+router.route("/user").get(getProfile);
```

Comment:
```suggestion
router.route("/user").get(protect, getProfile);
```
If we plan to obtain the logged in user's profile using this route - we might as well make it a protected one